static/templates/browse-companies.html

```diff
@@ -245,7 +245,7 @@ <h4>Messages</h4>
 										<!-- Notification -->
 										<li class="notifications-not-read">
 											<a href="dashboard-messages.html">
-												<span class="notification-avatar status-online"><img src="images/user-avatar-small-03.jpg" alt=""></span>
+												<span class="notification-avatar status-online"><img src="../../../../static/images/user-avatar-small-03.jpg" alt=""></span>

```

Comment:
this should likely just be ../../static not ../../../../static

Reply:
I used her original working index page as reference which had it that way. I just checked, seems to work with only '../../' so I adjusted all the img links to reflect that in another commit.